src/track/track.cpp

```diff
@@ -323,6 +324,18 @@ mixxx::ReplayGain Track::getReplayGain() const {
     return m_record.getMetadata().getTrackInfo().getReplayGain();
 }
 
+bool Track::setReplayGainDb(double ratio) {
+    QMutexLocker lock(&m_qMutex);
+    mixxx::ReplayGain replayGain = *m_record.refMetadata().refTrackInfo().ptrReplayGain();
```

Comment:
get/get/get

Reply:
I don't understand this comment

Reply:
mixxx::ReplayGain replayGain = m_record.getMetadata().getTrackInfo().getReplayGain();